site/docusaurus.config.ts

```diff
@@ -6,7 +6,8 @@ import GenAISamplesDocsPlugin from './src/plugins/genai-samples-docs-plugin';
 // This runs in Node.js - Don't use client-side code here (browser APIs, JSX...)
 
 const organizationName = 'openvinotoolkit';
```

Comment:
I would suggest taking both `organizationName` and `projectName` from env var:
```suggestion
// GITHUB_REPOSITORY env var is set by GitHub Actions
const GITHUB_REPOSITORY = process.env.GITHUB_REPOSITORY || 'openvinotoolkit/openvino.genai';
const [organizationName, projectName] = GITHUB_REPOSITORY.split('/');
```